src/main/python/rlbot/utils/process_configuration.py

```diff
@@ -116,8 +116,12 @@ def is_process_running(program, scriptname, required_args: Set[str]) -> Tuple[bo
                     return True, process
             except psutil.AccessDenied:
                 print(f"Access denied when trying to look at cmdline of {process}!")
-        # If we didn't return yet it means all matching programs were skipped.
-        raise WrongProcessArgs(f"{program} is not running with required arguments: {required_args}!")
+            except psutil.NoSuchProcess:
+                # process died whilst we were looking at it, so pretend we found none, it will get handles another time
+                break
```

Comment:
The other except block doesn't use break, it allows the loop through matching processes to continue. Is there a good reason to break here? For epic launch it'll probably be a list of one and not matter, but this fn might get used for other purposes also.

Reply:
By breaking here we get the same outcome to the function as if we had hit the same  race condition when checking the process name at line 102.

Reply:
I have pushed a better fix. The open question I have is whether the access denied should lead to the return False path or the WrongProcessArgs path. My thoughts are that if we have permission denied then it probably implies the process in question is being run by another user and as a result we should pretend it doesn't exist rather than it existing but being incorrect. Also the exception implies that the args for the found process are wrong and we cannot know that if we got access denied.

Reply:
Yep I agree.